packages/aws-appsync/src/client.js

```diff
@@ -106,7 +107,7 @@ class AWSAppSyncClient extends ApolloClient {
 
         let resolveClient;
 
-        const store = disableOffline ? null : createStore(() => this, () => resolveClient(this), conflictResolver);
+        const store = disableOffline ? null : createStore(() => this, () => resolveClient(this), persistOptions, conflictResolver);
```

Comment:
if you add persistOptions to the end of resolveClient's arg list this change would be backwards compatible..